.github/workflows/summarize-checks.yaml

```diff
@@ -0,0 +1,53 @@
+name: "[TEST-IGNORE] Summarize Checks"
+
+on:
+  workflow_run:
+    # todo, any others need to be added here?
+    workflows:
+      - "\\[TEST-IGNORE\\] Swagger SemanticValidation - Set Status"
+      - "\\[TEST-IGNORE\\] Swagger ModelValidation - Set Status"
+      - "Swagger Avocado - Set Status"
+      - "Swagger LintDiff - Set Status"
+      - "SDK Validation Status"
+    types: [completed]
+  pull_request_target: # when a PR is labeled. NOT pull_request, because that would run on the PR branch, not the base branch.
+    types:
```

Comment:
Probably want to run on opened/synchronized/reopened as well.  Example:

https://github.com/Azure/azure-rest-api-specs/blob/517eaf1cca58813605768f4ddc9a59ca75173493/.github/workflows/lintdiff-status.yaml#L10